spec/domain/queries/find_content_spec.rb

```diff
@@ -113,6 +113,36 @@
     end
   end
 
+  context 'when pagination parameters are provided' do
```

Comment:
Should we test results when . no pagination parameters are provided? We should return first page with default page size.

Reply:
Absolutely, That test was on the list but then I forgot to write it.